[2025-08-05T18:53:23.492+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-05T18:53:03.497985+00:00 [queued]>
[2025-08-05T18:53:23.529+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-05T18:53:03.497985+00:00 [queued]>
[2025-08-05T18:53:23.530+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-05T18:53:23.591+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): crea_vista_analitica> on 2025-08-05 18:53:03.497985+00:00
[2025-08-05T18:53:23.596+0000] {standard_task_runner.py:60} INFO - Started process 336 to run task
[2025-08-05T18:53:23.601+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'crea_vista_analitica', 'manual__2025-08-05T18:53:03.497985+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmpi5ajfe2x']
[2025-08-05T18:53:23.604+0000] {standard_task_runner.py:88} INFO - Job 219: Subtask crea_vista_analitica
[2025-08-05T18:53:23.702+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.crea_vista_analitica manual__2025-08-05T18:53:03.497985+00:00 [running]> on host 70eaa744a05c
[2025-08-05T18:53:23.874+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='crea_vista_analitica' AIRFLOW_CTX_EXECUTION_DATE='2025-08-05T18:53:03.497985+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-05T18:53:03.497985+00:00'
[2025-08-05T18:53:23.876+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-05T18:53:23.878+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/postgres_04_crea_vista_analitica.py']
[2025-08-05T18:53:23.886+0000] {subprocess.py:86} INFO - Output:
[2025-08-05T18:53:24.047+0000] {subprocess.py:93} INFO - Conectado a PostgreSQL en postgres:5432/medallon_db
[2025-08-05T18:53:24.048+0000] {subprocess.py:93} INFO - Ejecutando: /opt/airflow/sql/04_vista_analisis.sql
[2025-08-05T18:53:24.049+0000] {subprocess.py:93} INFO - Vista de an√°lisis creada/reemplazada correctamente
[2025-08-05T18:53:24.060+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-05T18:53:24.122+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=crea_vista_analitica, execution_date=20250805T185303, start_date=20250805T185323, end_date=20250805T185324
[2025-08-05T18:53:24.175+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-05T18:53:24.229+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
